src/planwise/client/current_project/handlers.cljs

```diff
@@ -48,20 +48,45 @@
     :transport :facilities-with-demand))
 
 (defn- project-loaded [db project-data]
-  (dispatch [:current-project/fetch-facility-types])
-  (dispatch [:regions/load-regions-with-geo [(:region-id project-data)]])
-  (db/new-viewmodel project-data))
+  (let [new-db (db/new-viewmodel project-data)]
+    (dispatch [:current-project/fetch-facility-types])
+    (dispatch [:regions/load-regions-with-geo [(:region-id project-data)]])
+    (if (:is-new? project-data)
+      (assoc-in new-db [:wizard :set] true)
+      new-db)))
```

Comment:
Right. My bad. Still, unless you are specifically testing with `false?` or something similar, `nil` and `false` are generally semantically equivalent.

Or... you can do:

```clojure
(assoc-in new-db [:wizard :set] (:is-new? project-data false))
```

if you really want a default value other than `nil`.